ci-operator/step-registry/baremetalds/e2e/upgrade/test/baremetalds-e2e-upgrade-test-commands.sh

```diff
@@ -0,0 +1,25 @@
+#!/bin/bash
+
+set -o nounset
+set -o errexit
+set -o pipefail
+
+echo "************ baremetalds e2e upgrade test command ************"
+#export TEST_PROVIDER=packet
+
+if [[ -n "${TEST_OPTIONS}" ]]; then
+    TEST_ARGS="--options=${TEST_OPTIONS}"
+fi
+
+if [[ -n "${TEST_SKIPS}" ]]; then
+    TESTS="$(openshift-tests "${TEST_COMMAND}" --dry-run "${TEST_SUITE}")"
+    echo "${TESTS}" | grep -v "${TEST_SKIPS}" >/tmp/tests
+    echo "Skipping tests:"
+    echo "${TESTS}" | grep "${TEST_SKIPS}"
+    TEST_ARGS="${TEST_ARGS:-} --file /tmp/tests"
+fi
+
```

Comment:
```suggestion

if -n "${OPENSHIFT_UPGRADE_RELEASE_IMAGE_OVERRIDE}" ]]; then
    # OPENSHIFT_UPGRADE_RELEASE_IMAGE_OVERRIDE is a pullspec of release:latest imagestreamtag
    TEST_ARGS="${TEST_ARGS:-} --to-image=${OPENSHIFT_UPGRADE_RELEASE_IMAGE_OVERRIDE}"
fi

```
See https://github.com/openshift/release/blob/master/ci-operator/step-registry/openshift/e2e/test/openshift-e2e-test-commands.sh#L73-L76